src/main/java/seedu/address/model/pet/Color.java

```diff
@@ -17,7 +15,7 @@ public class Color implements Comparable<Color> {
      * @param value The string representation of a color.
      */
     public Color(String value) {
-        this.value = Objects.requireNonNullElse(value, "");
+        this.value = (value == null) || !value.matches("[a-zA-Z][a-zA-Z ]+") ? "" : value;
```

Comment:
According to our UG, color could contain numbers. I don't think we restrict any attribute to be letters only. Can we stick to the regex in the `Name` class?
`public static final String VALIDATION_REGEX = "[\\p{Alnum}][\\p{Alnum} ]*";`

Reply:
i changed it to ^[a-zA-Z0-9 ]*$ which seems to work the 0-9 part accepts numeric input